bitnami/grafana/values.yaml

```diff
@@ -686,7 +686,31 @@ metrics:
     ## @param metrics.serviceMonitor.jobLabel The name of the label on the target service to use as the job name in prometheus.
     ##
     jobLabel: ""
-
+  ## Prometheus Operator PrometheusRule configuration
+  ##
+  prometheusRules:
+    ## @param metrics.prometheusRules.enabled if `true`, creates a Prometheus Operator PrometheusRule (also requires `metrics.enabled` to be `true`, and makes little sense without ServiceMonitor)
+    ##
+    enabled: false
+    ## @param metrics.prometheusRules.selector [object] Additional labels to add to the PrometheusRule so it is picked up by the operator
+    ## If using the [Helm Chart](https://github.com/helm/charts/tree/master/stable/prometheus-operator) this is the name of the Helm release and 'app: prometheus-operator'
+    ##
+    selector:
+      app: prometheus-operator
+      release: prometheus
+    ## @param metrics.prometheusRules.rules PrometheusRule rules to configure
+    ## e.g:
+    ##  - alert: Grafana-Down
+    ##    annotations:
+    ##      message: 'Grafana instance is down'
+    ##      summary: Grafana instance is down
+    ##    expr: absent(up{job="grafana"} == 1)
+    ##    labels:
+    ##      severity: warning
+    ##      service: grafana
+    ##    for: 5m
+    ##
+    rules: {}
```

Comment:
Could you follow the same format we use in other charts (e.g. Apache) for this kind of objects? See https://github.com/bitnami/charts/blob/master/bitnami/apache/values.yaml#L517

Reply:
Sure! Thank you!